### PR TITLE
Upgrade Connext DDS version in master to 6.1.1

### PR DESCRIPTION
**Pull request by fgaranda**
### Summary
This PR upgrades the version master targets to Connext DDS 6.1.1.

### Details and comments

As part of this PR, we upgrade schema files, build scripts, and other infrastructure in the master branch to use Connext DDS 6.1.1.

### Checks

-   [x] I have updated the documentation accordingly.
-   [x] I have read the [CONTRIBUTING](https://github.com/rticommunity/rticonnextdds-examples/blob/develop/CONTRIBUTING.md) document.

### CODE REVIEW

File: examples/connext_dds/dynamic_data_using_publisher_subscriber/cs/Shapes.csproj [hunk 1]
```diff
@@ -6,8 +6,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Rti.ConnextDds.Extra" Version="6.1.0" />
```
Comment (alexcamposruiz) [inline examples/connext_dds/dynamic_data_using_publisher_subscriber/cs/Shapes.csproj:9]: The search/replace is wrong. It removed .Extra in many .csprojs.

**Review by alexcamposruiz — CHANGES_REQUESTED** [review]

**Comment by ManuelJNunez** [comment]
We are currently upgrading the Connext Version of the bundle used for CI. Once it is ready we can restart the CI workflow.

**Comment by ManuelJNunez** [comment]
Also, I'm going to temporarily deactivate the linting for .NET.

**Review by alexcamposruiz — APPROVED** [review]

**Comment by lulivi** [comment]
Waiting for the build to end.

**Review by ManuelJNunez — APPROVED** [review]

**Pull request merged**